test/functional/wallet_createwallet.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test createwallet arguments.
+"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_raises_rpc_error,
+)
+
+class CreateWalletTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = False
+        self.num_nodes = 1
+        self.supports_cli = True
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        node = self.nodes[0]
+        node.generate(1) # Leave IBD for sethdseed
+
+        self.nodes[0].createwallet(wallet_name='w0')
+        w0 = node.get_wallet_rpc('w0')
+        address1 = w0.getnewaddress()
+
+        self.log.info("Test disableprivatekeys creation.")
+        self.nodes[0].createwallet(wallet_name='w1', disable_private_keys='true')
```

Comment:
@jnewbery this test fails in `--usecli` if I use a boolean `True` rather than a string `'true'`. Is that by design?